doc/esmvalcore/datafinder.rst

```diff
@@ -1,7 +1,251 @@
+configuration file ``config-user.yml``. The two sections where the user will
+set the paths are ``rootpath`` and ``drs``. ``rootpath`` contains pointers to
+``CMIP``, ``OBS``, ``default`` and ``RAWOBS`` root paths; ``drs`` sets the type
+of directory structure the root paths are structured by. It is important to
+first discuss the ``drs`` parameter: as we've seen in the previous section, the
+DRS as a standard is used for both file naming conventions and for directory
+structures. 
+
+.. _config-user-drs:
+
+Explaining ``config-user/drs: CMIP5:`` or ``config-user/drs: CMIP6:``
+---------------------------------------------------------------------
+Whreas ESMValTool will **always** use the CMOR standard for file naming (please
+refer above), by setting the ``drs`` parameter the user tells the tool what
+type of root paths they need the data from, e.g.: 
+
+  .. code-block:: yaml
+
+   drs:
+     CMIP6: BADC
+
+will tell the tool that the user needs data from a repository structured
+according to the BADC DRS structure, i.e.:
+
+``ROOT/[institute]/[dataset_name]/[experiment]/[ensemble]/[mip]/[variable_short_name]/[grid]``;
+
+setting the ``ROOT`` parameter is explained below. This is a
+strictly-structured repository tree and if there are any sort of irregularities
+(e.g. there is no ``[mip]`` directory) the data will not be found! ``BADC`` can
+be replaced with ``DKRZ`` or ``ETHZ`` depending on the existing ``ROOT``
+directory structure.
+The snippet
+
+  .. code-block:: yaml
+
+   drs:
+     CMIP6: default
+
+is another way to retrieve data from a ``ROOT`` directory that has no DRS-like
+structure; ``default`` indicates that the data lies in a directory that
+contains all the files without any structire.
+
+.. note::
+   When using ``CMIP6: default`` or ``CMIP5: default`` it is important to
+   remember that all the needed files must be in the same top-level directory
+   set by ``default`` (see below how to set ``default``).
+
+.. _config-user-rootpath:
+
+Explaining ``config-user/rootpath:``
+------------------------------------
+
+``rootpath`` identifies the root directory for different data types (``ROOT`` as we used it above):
+
+* ``CMIP`` e.g. ``CMIP5`` or ``CMIP6``: this is the `root` path(s) to where the
+  CMIP files are stored; it can be a single path or a list of paths; it can
+  point to an ESGF node or it can point to a user private repository. Example
+  for a CMIP5 root path pointing to the ESGF node on CEDA-Jasmin (formerly
+  known as BADC):
+
+  .. code-block:: yaml
+
+    CMIP5: /badc/cmip5/data/cmip5/output1
+
+  Example for a CMIP6 root path pointing to the ESGF node on CEDA-Jasmin:
+
+  .. code-block:: yaml
+
+    CMIP6: /badc/cmip6/data/CMIP6/CMIP
+
+  Example for a mix of CMIP6 root path pointing to the ESGF node on CEDA-Jasmin
+  and a user-specific data repository for extra data:
+
+  .. code-block:: yaml
+
+    CMIP6: [/badc/cmip6/data/CMIP6/CMIP, /home/users/joepesci/cmip_data]
```

Comment:
joe pesci is working with CMIP data??  :D

Reply:
Not anymore, removed.